src/docs/blogs/test.md

```diff
@@ -0,0 +1,9 @@
+# Title: This is a sample blog
+
+# Author: John
+
+# Date: 4.4.17
+
+### Content:
```

Comment:
This syntax is valid markdown, and thus seems like it would get rendered along with the content. The JSON/YAML soultion might work better.